jca-and-tomcat/repo/com/github/fungal/fungal/0.11.0.Final/fungal-0.11.0.Final.pom.sha1

```diff
@@ -0,0 +1 @@
+e91f98a175adf47dcc1baf86d8134888db61ee75
```

Comment:
How do you calculate this ID

Reply:
as I've already mentioned on in chat the id is `sha1` hash of the file. You can verify if it's placed correctly by running
`sha1sum jca-and-tomcat/repo/com/github/fungal/fungal/0.11.0.Final/fungal-0.11.0.Final.pom`